timesketch/frontend-ng/src/views/Studio.vue

```diff
@@ -0,0 +1,69 @@
+<!--
+Copyright 2022 Google Inc. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+  <div>
+    <!-- Left panel -->
+    <v-navigation-drawer app permanent :width="navigationDrawer.width" hide-overlay ref="drawer">
+      <v-toolbar flat color="transparent">
+        <v-avatar class="ml-n1">
+          <router-link to="/">
+            <v-img src="/dist/timesketch-color.png" max-height="25" max-width="25" contain></v-img>
+          </router-link> </v-avatar
+        ><span style="font-size: 1.2em">Studio</span>
+        <v-spacer></v-spacer>
+      </v-toolbar>
+      <v-divider></v-divider>
+      <ts-sigma-rules></ts-sigma-rules>
+    </v-navigation-drawer>
+    <ts-sigma-rule-modification app @cancel="formatXMLString = false" :rule_uuid="id" v-if="type === 'sigma'">
```

Comment:
This need to be refactored: Studio.vue should be the "Base view" with the left panel rendered, and then a v-router for the other pages: `SigmaStudio.vue` for now, `SearchTemplateStudio.vue` and `ScenarioStudio.vue` in the future.

Look at the `Sketch.vue` and `Explore.vue` for an example

Reply:
That would just be renaming TsSigmaRuleModification to TsSigmaStudio, no?

Reply:
We will need to do this in a followup PR.